webapp/src/components/data_selectors/jira_autocomplete_selector/jira_autocomplete_selector.tsx

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import AsyncSelect from 'react-select/async';
+import debounce from 'debounce-promise';
+
+import Setting from 'components/setting';
+import {getStyleForReactSelect} from 'utils/styles';
+import {ReactSelectOption} from 'types/model';
+
+import BackendSelector, {Props as BackendSelectorProps} from '../backend_selector';
+
+const searchDebounceDelay = 400;
+
+const stripHTML = (text: string) => {
+    if (!text) {
+        return text;
+    }
+
+    var doc = new DOMParser().parseFromString(text, 'text/html');
+    return doc.body.textContent || '';
+ }
+
+type Props = BackendSelectorProps & {
+    searchAutoCompleteFields: (params: object) => Promise<any>;
```

Comment:
Not real fan of anys, objects, {}, or any other "generic" types on typescript. Is there any chance to use specific types?

Reply:
Thanks for catching this. I did not thoroughly go through the types from the labels PR. This should be straightforward to fix.